src/test/ui/parser/removed-syntax-field-let.stderr

```diff
@@ -1,10 +1,15 @@
 error: expected identifier, found keyword `let`
   --> $DIR/removed-syntax-field-let.rs:2:5
    |
-LL | struct S {
-   |        - while parsing this struct
 LL |     let foo: (),
    |     ^^^ expected identifier, found keyword
+   |
+   = help: see <https://doc.rust-lang.org/book/ch05-01-defining-structs.html> for more information
+help: remove the let, the `let` keyword is not allowed in struct field definitions
```

Comment:
Not sure how to fix this but the formatting of these 2 help messages seems a little off and not uniform with other diagnostics IMO. Maybe the top line can be moved to the last line, below the suggested removal of `let`, if that is possible.

Reply:
Take a peek at the simplification performed later on with Est's review.